.github/workflows/run-tests.yml

```diff
@@ -280,6 +280,15 @@ jobs:
         if: ${{ !cancelled() }}
         run: make upload-test-results
```

Comment:
can we get rid of this buildkite stuff now?

Reply:
I tweaked the steps added in this PR a bit and got rid of the buildkite stuff in #6668.